### DISCUSSION TITLE
how to work cron job seconds time?

### DISCUSSION THREAD

**Discussion by Otiss2627** [supabase/supabase#15916] [Questions]
Example:

CREATE TABLE public.test_cron (
  id bigint generated by default as identity,
  pg12 timestamp,
  primary key(id)
);

select cron.schedule('*/30 * * * * *','insert into test_cron (pg12) values (now())');

OR 

select cron.schedule('* * * * * *','insert into test_cron (pg12) values (now())');

And I can't work with seconds in cron

**Comment by GaryAustin1** [comment]
The current cron version Supabase uses only supports minutes.  There was an issue with the latest cron version (which does support seconds) and they backed it out.
![image](https://github.com/supabase/supabase/assets/54564956/096714fa-b396-4e5d-a752-9d587d068f66)
![image](https://github.com/supabase/supabase/assets/54564956/4047bbb7-6e7c-44d3-b655-90971ca075c2)
Note the dates.

**Comment by anil1kuppa** [comment]
Any plan to add seconds too in the supabase cron?

**Reply by GaryAustin1** [reply]
Pretty sure that has been supported for quite awhile now.
